tests/integration.rs

```diff
@@ -120,6 +120,25 @@ fn invert() -> Result<()> {
   assert_output_eq(&["resize:1:1", "all"], "1")
 }
 
+#[test]
+fn save() -> Result<()> {
+  let path = "output.txt";
```

Comment:
I mentioned this in the other PR, but I don't think we should save and load the .txt format. It can be an unstable format that we use a dedicated command like "print" to show.